src/main/java/org/openrewrite/kotlin/tree/K.java

```diff
@@ -708,30 +725,138 @@ public List<Modifier> getModifiers() {
         }
 
         @Nullable
+        @With
+        @Getter
         JRightPadded<NameTree> receiver;
 
-        @Override
-        public @Nullable JavaType getType() {
-            return typedTree.getType();
+        @Nullable
+        JContainer<TypeTree> parameters;
```

Comment:
Note: This is `JContainer<TypeTree>` rather than `JContainer<Parameter>` to allow for `J.Empty` when empty.